parts/kubernetesagentvars.t

```diff
@@ -4,6 +4,16 @@
     "{{.Name}}StorageAccountsCount": "[add(div(variables('{{.Name}}Count'), variables('maxVMsPerStorageAccount')), mod(add(mod(variables('{{.Name}}Count'), variables('maxVMsPerStorageAccount')),2), add(mod(variables('{{.Name}}Count'), variables('maxVMsPerStorageAccount')),1)))]",
     "{{.Name}}VMNamePrefix": "[concat(variables('orchestratorName'), '-{{.Name}}-', variables('nameSuffix'), '-')]", 
     "{{.Name}}VMSize": "[parameters('{{.Name}}VMSize')]",
+{{if .IsWindows}}
+    "{{.Name}}IPAddressName": "[concat(variables('orchestratorName'), '-agent-ip-', variables('{{.Name}}Index'), '-', variables('nameSuffix'))]",
+    "{{.Name}}LbBackendPoolName": "[concat(variables('orchestratorName'), '-{{.Name}}-', variables('nameSuffix'))]", 
+    "{{.Name}}LbID": "[resourceId('Microsoft.Network/loadBalancers',variables('{{.Name}}LbName'))]", 
```

Comment:
Why are we doing any of this? I'd prefer:

1. NAT to the RDP port for the Windows nodes (but if you do this, expose the normal agents over NAT as well...)

2. We continue to not expose agents to the Internet and instead tell people to SSH port forward to reach the Windows nodes.